build-for-compare.py

```diff
@@ -306,11 +308,12 @@ def main():
             logger.info('Running configure script')
             opt = shell_join(args.opt)
             check_call(['./configure', '--disable-hardening', '--with-incompatible-bdb', '--without-cli', '--disable-tests', '--disable-ccache',
+                '--prefix={}'.format(args.prefix) if args.prefix else '',
```

Comment:
Have just realised I should update this so that `--with-incompatible-bdb` is only passed in the non-depends case. Will change shortly.